components/installer/README.md

```diff
@@ -120,12 +120,20 @@ Helm installation.
 
 ## Install selected components only
 
-By default, this tool installs a predefined component package. If necessary, you can specify which components to install by customizing the component list located in the [installer-cr.yaml.tpl](../../installation/resources/installer-cr.yaml.tpl) file. Available components:
-
-- cluster-essentials
-- istio
-- prometheus-operator
-- provision-bundles
-- dex
-- core
-- remote-environments
\ No newline at end of file
+This tool installs components specified in installation CR. Please see CR template in the [installer-cr.yaml.tpl](../../installation/resources/installer-cr.yaml.tpl) file for more details. 
+
+To enable installation of a component, specify its name and Namespace. If you want the release name to be different from the component's name, provide the release parameter.
+
+Example:
+
+```
+spec:
+    ...
+    components:
+    ...
+    - name: "remote-environments"
+      namespace: "kyma-integration"
+      release: "hmc-default"
+```
+
+In the example above, the `remote-environments` component will be installed in the `kyma-integration` Namespace, using `hmc-default` as the release name.. Fields `name` and `namespace` are mandatory. The name of the component is also the name of the component subdirectory in the `resources` directory. Installer assumes that the component subdirectory is a valid Helm chart.
```

Comment:
1. `In the example` ~above~``, the `remote-environments` component is installed``...
Also: please remove double dot at the end of this sentence.

2. ``The **name** and **namespace** fields are mandatory.``

3. `The Installer assumes`...

Reply:
Done